### PR TITLE
Make only first letter of notes upper case

### LINKED ISSUE #1: Cannot parse note "eb3"
@justinjmoses What am I doing wrong here?

```
node-keyboard>sequence(c, 2, 1, 2, 2, 1, 2, 2)
[ 'c3', 'd3', 'eb3', 'f3', 'g3', 'g#3', 'a#3', 'c4' ]
node-keyboard>sequence(c, 2, 1, 2, 2, 1, 2, 2).map(play)

node_modules/node-keyboard/lib/mappers.js:14
        throw new Error(`Cannot parse note "${chunk.input}"\n`)
        ^
Error: Cannot parse note "eb3"

    at exports.play.e (node_modules/node-keyboard/lib/mappers.js:14:15)
    at Array.map (native)
    at evalmachine.<anonymous>:1:34
    at Object.exports.runInContext (vm.js:44:17)
    at REPLServer.module.exports (node_modules/node-keyboard/lib/evaluator.js:28:23)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.<anonymous> (repl.js:441:10)
    at emitOne (events.js:96:13)
    at REPLServer.emit (events.js:188:7)
node-keyboard>play('eb3');
play('eb3');
Error: Cannot parse note "eb3"

    at exports.play.e (node_modules/node-keyboard/lib/mappers.js:14:15)
    at evalmachine.<anonymous>:1:1
    at Object.exports.runInContext (vm.js:44:17)
    at REPLServer.module.exports (node_modules/node-keyboard/lib/evaluator.js:28:23)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.<anonymous> (repl.js:441:10)
    at emitOne (events.js:96:13)
    at REPLServer.emit (events.js:188:7)
    at REPLServer.Interface._onLine (readline.js:224:10)
```

### PR DESCRIPTION
**Pull request by winniehell**
fixes #1

### CODE REVIEW

**Comment by winniehell** [comment]
@justinjmoses checks green! :smile:

File: lib/instruments.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ module.exports = {
         const { input, instrument = 'acoustic_grand_piano' } = chunk
 
         function pathToAudio(note) {
-            return path.join(pathToSoundfonts, `${instrument}-mp3`, `${note.toString().toUpperCase()}.mp3`)
+            const noteString = note.toString();
+            return path.join(pathToSoundfonts, `${instrument}-mp3`, `${noteString.charAt(0).toUpperCase() + noteString.slice(1)}.mp3`)
```
Comment (jjgonecrypto) [inline lib/instruments.js]: What about `${note.toString().replace(/^./, c => c.toUpperCase())}.mp3`?

**Review by jjgonecrypto — CHANGES_REQUESTED** [review]
Could you use the simpler version?

File: lib/instruments.js (hunk 1 above)
Comment (jjgonecrypto) [reply lib/instruments.js]: Actually, that won't work for `play(eB)`. Think we need to `toString().toLowerCase().replace(...)`

**Comment by winniehell** [comment]
Makes sense! Thanks for the comment! :+1: 

Successfully tested with:

```
node-keyboard>play(eB3);
{ input: 'eb3',
  kill: [Function],
  played: 'acoustic_grand_piano: 𝄞 ♩ eb3' }
node-keyboard>play(Eb3);
{ input: 'eb3',
  kill: [Function],
  played: 'acoustic_grand_piano: 𝄞 ♩ eb3' }
node-keyboard>play(eb3);
{ input: 'eb3',
  kill: [Function],
  played: 'acoustic_grand_piano: 𝄞 ♩ eb3' }
node-keyboard>play(EB3);
{ input: 'eb3',
  kill: [Function],
  played: 'acoustic_grand_piano: 𝄞 ♩ eb3' }
```

**Review by jjgonecrypto — APPROVED** [review]

**Pull request merged**